README.md

```diff
@@ -1,5 +1,7 @@
-# Mix Template—a package for creating customized directory trees
+# Mix Template — Create Customized Directory Trees
 
```

Comment:
There shouldn't be spaces around the em-dash. Also, I think I prefer the low case (it's kind of a subheading). so

$ Mix Template—create customized directory trees

Reply:
Ok.